contrib/go-pg/pg/example_test.go

```diff
@@ -0,0 +1,40 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
We need the folder name to be `contrib/go-pg/pg.v10/...` as described here: https://github.com/DataDog/dd-trace-go/tree/v1/contrib#usage

Reply:
How do you think i should rename package name? Let `pg` or rename to for eg: `pgv10`

Reply:
Good question. We should keep in named `pg`. We do the same thing for grpc and a few others.
I will update the documentation to clarify that.